nxsdk_modules_contrib/pelenet/README.md

```diff
@@ -0,0 +1,104 @@
+# PeleNet - Reservoir computing framework for Loihi
+
+Pele is the goddess of volcanoes and fire in the Hawaiian religion. She has therefore also the control over Loihi.
+
+The PeleNet is a framework for reservoir computing. It has the following features:
+
+* Defining experiments, which can contain one or multiple network simulations
+* Selecting and preprocessing of probes (including readout)
+* Extensive management of parameters (default and specific for an experiment)
+* Connectvity matrix abstraction (connectivity matrix is automatically splitted and ditributed over the Loihi cores)
+* Logging of experiments
+* Object-oriented code design
+
+## Warning
+
+The framework is still under development. Some functionality may not work as described. For questions please contact me (see below).
+
+## Using the framework
+
+### Prerequisites
+
+The framework is based on the NxSDK from Intel in version 0.9.x.
```

Comment:
Could you also add the particular version of nxsdk you developed/tested against so that we can install that particular release to ensure that ipython notebooks run without any issue.